src/litdata/processing/data_processor.py

```diff
@@ -1019,7 +1019,7 @@ def run(self, data_recipe: DataRecipe) -> None:
         print("Workers are finished.")
         result = data_recipe._done(len(user_items), self.delete_cached_files, self.output_dir)
 
-        if num_nodes == node_rank + 1 and self.output_dir.url and _IS_IN_STUDIO and self.input_dir.path:
+        if num_nodes == node_rank + 1 and self.output_dir.url and _IS_IN_STUDIO:  # and self.input_dir.path:
```

Comment:
I think we should add (self.output_dir.path && self.output_dir.startswith("/teamspace/datasets"))

Reply:
couldn't the output dir also be an s3 url? Isn't the output_dir.url already resolved to the S3 location, even when it is a `teamspace/datasets` path?

Reply:
Yes, but I think this would be true if you provided the path to an s3 connection and create an empty dataset